### ISSUE TITLE
make RASPBERRY_PI_IP configurable

### ISSUE THREAD

**Issue by WolfgangFahl** [rc-dukes/dukes#7]
The Raspberry PI's IP address 10.9.8.7 is hard encoded in several places 
e.g. 

1. Environment.java
2. deploy.sh
3. BoarsNest.java

There should be a single place for configuration of the ip address.

**Comment by WolfgangFahl** [comment]
The java script in boss-hog/src/main/resources/web/index.html still has a 10.9.8.7 entry:
```
 function updateCamConfig(elementId, configKey) {
                var configValue = document.getElementById(elementId).value;
                var url = 'http://10.9.8.7/html/cmd_pipe.php?cmd=' + configKey + '%20' + configValue;

                var ajax_cmd = new XMLHttpRequest();
                ajax_cmd.open("GET",url,true);
                ajax_cmd.send();
            }
```

**Comment by bertjan** [comment]
We could expose the Raspberry Pi IP via a REST endpoint or Vertx eventbus endpoint in the backend. Upon startup, the frontend can grab the ip there and configure the video URL dynamically.

**Comment by WolfgangFahl** [comment]
Mostly fixed #43 has the leftover task.
